csm-extensions/services/dev-mysql/chart/values.yaml

```diff
@@ -0,0 +1,24 @@
+---
+env:
+  CF_ADMIN_PASSWORD: ~
+  CF_ADMIN_USER: ~
+  CF_CA_CERT: ~
+  CF_DOMAIN: ~
+  SERVICE_LOCATION: ~
```

Comment:
If not set, we can use the internal kube thing (`<name>.<namespace>.svc.<kube service doman>`), right?

Reply:
I do not know `helm` enough to say one way or other.